src/Sylius/Bundle/AdminBundle/Resources/views/Crud/Create/_content.html.twig

```diff
@@ -7,8 +7,10 @@
 <div class="ui segment">
     {{ form_start(form, {'action': path(configuration.vars.route.name|default(configuration.getRouteName('create')), configuration.vars.route.parameters|default({})), 'attr': {'class': 'ui loadable form', 'novalidate': 'novalidate'}}) }}
     {% if configuration.vars.templates.form is defined %}
+        {% include '@SyliusAdmin/form_validation_errors_checker.html.twig' %}
```

Comment:
Is there any reason, why we have to declare it twice? Won't it work properly, if we extract include before if statement?